LayoutTests/imported/w3c/web-platform-tests/subresource-integrity/integrity-policy/script.https-expected.txt

```diff
@@ -1,16 +1,14 @@
 
 
-Harness Error (TIMEOUT), message = null
-
-TIMEOUT Ensure that a script without integrity did not run Test timed out
-NOTRUN Ensure that a script with unknown integrity algorithm did not run
-NOTRUN Ensure that a script without integrity algorithm runs and gets reported in report-only mode
-NOTRUN Ensure that a no-cors script gets blocked
-NOTRUN Ensure that ReportingObserver gets called without endpoints
-NOTRUN Ensure that a script with integrity runs
-NOTRUN Ensure that a script with signature integrity runs
-NOTRUN Ensure that a data URI script with no integrity runs
-NOTRUN Ensure that a no-CORS data URI script with no integrity runs
-NOTRUN Ensure that a blob URL script with no integrity runs
-NOTRUN Ensure that a no-CORS blob URL script with no integrity runs
+PASS Ensure that a script without integrity did not run
+PASS Ensure that a script with unknown integrity algorithm did not run
+PASS Ensure that a script without integrity algorithm runs and gets reported in report-only mode
+PASS Ensure that a no-cors script gets blocked
+PASS Ensure that ReportingObserver gets called without endpoints
+PASS Ensure that a script with integrity runs
+FAIL Ensure that a script with signature integrity runs assert_equals: expected true but got false
```

Comment:
Is signature integrity a separate specification? Is that planned?

Reply:
It is a separate spec, and this test case should've been a tentative test. I can work on separating that upstream.